docs/pages/router/reference/link-preview.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Link Preview
+description: Learn how to add a preview to your link on iOS.
+hasVideoLink: false
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { FileTree } from '~/ui/components/FileTree';
+
+<ContentSpotlight
+  alt="A video presenting the link preview feature in Expo Router."
+  file="/expo-router/link-preview.mp4"
+/>
+
+This guide will show you how to add and customize a link preview for your app on iOS. A link preview is a feature commonly used on iOS to show user a preview of the content linked by the link.
```

Comment:
This should mention the iOS name of the feature: Peek and Pop https://developer.apple.com/documentation/UIKit/implementing-peek-and-pop